flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/DispatcherFailoverITCase.java

```diff
@@ -109,12 +114,16 @@ public void testRecoverFromCheckpointAfterJobGraphRemovalOfTerminatedJobFailed()
         final JobID jobId = jobGraph.getJobID();
 
         // Construct job graph store.
-        final Error jobGraphRemovalError = new Error("Unable to remove job graph.");
+        final Error temporaryError = new Error("Unable to remove job graph.");
+        final AtomicReference<? extends Error> temporaryErrorRef =
```

Comment:
Why error instead of a exception?

Reply:
I guess, the initial intention was to have `Error` as the selection condition in [ExceptionUtils.findThrowable(throwable, Error.class)](https://github.com/apache/flink/blob/master/flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/DispatcherFailoverITCase.java#L134) further down in the the test code. I left it like that because it doesn't cause any harm...